DEPS

```diff
@@ -15,6 +15,8 @@ deps = {
   "vendor/bip39wally-core-native": "https://github.com/brave-intl/bip39wally-core-native.git@9b119931c702d55be994117eb505d56310720b1d",
   "vendor/bat-native-anonize": "https://github.com/brave-intl/bat-native-anonize.git@0559543f458a949b83b58035273ef7f8f1a1b111",
   "vendor/bat-native-tweetnacl": "https://github.com/brave-intl/bat-native-tweetnacl.git@1b4362968c8f22720bfb75af6f506d4ecc0f3116",
+  "components/brave_sync/extension/brave-sync": "https://github.com/brave/sync.git@75c1bb10c4a54275de4fb10afd9f204cc07284a8",
```

Comment:
note that this will use the sync staging server, not prod

Reply:
OK, for production I need to use another commit has for brave-sync DEPS and to pull server url out from 
https://github.com/brave/brave-core/pull/226/files#diff-29acb92f207e09cdd1b1ea416ac0767bR417
into the build parameter